src/global_logic.js

```diff
@@ -1227,6 +1238,7 @@ module.exports.getTotalBuff = function (prof) {
     totalBuff["zenithChainDamageLimit"] += zenithChainDamageLimit[prof.zenithChainDamageLimitBonus] != undefined ? zenithChainDamageLimit[prof.zenithChainDamageLimitBonus] : 0;
     totalBuff["zenithElement"] += zenithElement[prof.zenithElementBonus] != undefined ? zenithElement[prof.zenithElementBonus] : 0;
     totalBuff["zenithDamageLimit"] += zenithDamageLimit[prof.zenithDamageLimitBonus] != undefined ? zenithDamageLimit[prof.zenithDamageLimitBonus] : 0;
+    totalBuff["criticalBuff"] = prof.criticalBuff != undefined ? prof.criticalBuff : [];
```

Comment:
(Not change request, just coding info)
`totalBuff["criticalBuff"] = prof.criticalBuff || [];`

could be short way (if criticalBuff was non-number,non-string object)
however, other code are checking undefined, it's ok follow legacy code rule.
(if change those all legacy code, I want refactoring topic, because that make extra tests costs)